src/components/Image.tsx

```diff
@@ -0,0 +1,13 @@
+import NextImage, { getImageProps } from 'next/image';
+import { ComponentProps } from 'react';
+
+export default function Image(props: ComponentProps<typeof NextImage>) {
+  const { props: nextProps } = getImageProps({
+    ...props,
+  });
+
+  // eslint-disable-next-line no-unused-vars
+  const { style: _omit, ...delegated } = nextProps;
```

Comment:
[question] Is this equivalent to `const { style, ...delegated } = nextProps`, i.e. the `style` key/value is being removed from the object?

Reply:
Yeah, I was following the workaround provided on a [related issue](https://github.com/vercel/next.js/issues/45184#issuecomment-1988319088), but now that I'm looking at it I'm not sure why `_omit` would be necessary, I can update to remove that.

Reply:
No need to change, was just curious. I kind of like the explicitness of `_omit`.